<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zadania</class>
 <widget class="QWidget" name="Zadania">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Zakres czasowy</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="2">
           <widget class="QDateEdit" name="dateEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Od:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Do:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDateEdit" name="dateEdit_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Uwzględnij zakres czasowy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="checkBoxUwzglednijDate">
            <property name="text">
             <string>Tak/Nie</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonWyszukajPoKryteriach">
       <property name="text">
        <string>Wyszukaj według kryteriów</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QTableWidget" name="tableWidgetZadania"/>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="title">
        <string>Wyszukaj według</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBoxMiejscowosci">
              <property name="text">
               <string>Miejscowości </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxMiejscowosci">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QCheckBox" name="checkBoxZespoly">
              <property name="text">
               <string>Zespołu</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxZespoly"/>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="checkBoxStacja">
              <property name="text">
               <string>Stacji</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxStacja">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonWyszukajWszystko">
       <property name="text">
        <string>Wyszukaj wszystkie zadania</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
